docker/.env.example

```
@@ -90,6 +90,7 @@ WEBHOOK_SECRET=
 # Keep it stable — rotating it
 # requires re-encrypting every connection row. Generate: `openssl rand -base64 24 | cut -c1-32`
 HEPHAESTUS_SECURITY_ENCRYPTION_KEY=
+HEPHAESTUS_SECURITY_CREDENTIAL_ENCRYPTION_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HEPHAESTUS_SECURITY_CREDENTIAL_ENCRYPTION_KEY key should go before the HEPHAESTUS_SECURITY_ENCRYPTION_KEY key

(UnorderedKey)